beesdoo_easy_my_coop/models/res_partner.py

```diff
@@ -38,7 +32,7 @@ def _cooperator_share_type(self):
         'share_ids.share_product_id.default_code',
         'share_ids.share_number',
```

Comment:
simpler to set the depedency on cooperator_type fields as this is what interested us. And that computation of cooperator_type is triggered on those fields.